curvefs/src/mds/metaserverclient/metaserver_client.cpp

```diff
@@ -274,6 +274,8 @@ FSStatusCode MetaserverClient::CreateDentry(
     d->set_txid(0);
     d->set_type(FsFileType::TYPE_DIRECTORY);
     request.set_allocated_dentry(d);
+    request.set_time(0);
```

Comment:
It may be more appropriate to use real time

Reply:
ok